### PR TITLE
feat:adding toast to CL

### PR DESCRIPTION
**Pull request by mariyasada**
adding toast feature to CL. i have added the toast link to all the files. do review it

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: htmlFiles/toast.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+                        <li class="sidebar-list-item sidebar-component-list component-active">
+                            <a href="/htmlFiles/toast.html">Toast</a>
+                        </li>
+
+          </ul>
+      </div>
+<!--------------------------------  side bar ends here-------------main content starts here--------------------------------- -->
+      <section class="main-content-container">
+                <div class="section1-container">
```
Comment (ShreyashMahajan) [inline htmlFiles/toast.html:122]: instead of number, try to give name

**Review by ShreyashMahajan — COMMENTED** [review]
the naming of code is perfect, and properties are also used very well. The code was really very good

File: htmlFiles/toast.html (hunk 1 above)
Comment (mariyasada) [reply htmlFiles/toast.html:122]: I will try next time.

**Review by ShreyashMahajan — APPROVED** [review]

**Pull request merged**